src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -92,6 +93,43 @@ def load_arguments(self, _):
                    help="Create your Azure Spring Cloud service in an Azure availability zone or not, "
                         "this could only be supported in several regions at the moment.",
                    default=False, is_preview=True)
+        c.argument('enable_application_configuration_service',
+                   arg_type=get_three_state_flag(),
+                   default=False,
+                   is_preview=True,
+                   options_list=['--enable-application-configuration-service', '--enable-acs'],
+                   help='(Enterprise Tier Only) Enable Application Configuration Service.')
+        c.argument('enable_service_registry',
+                   arg_type=get_three_state_flag(),
+                   default=False,
+                   is_preview=True,
+                   options_list=['--enable-service-registry', '--enable-sr'],
+                   help='(Enterprise Tier Only) Enable Service Registry.')
+        c.argument('enable_gateway',
+                   arg_group="Spring Cloud Gateway",
+                   arg_type=get_three_state_flag(),
+                   default=False,
+                   is_preview=True,
+                   help='(Enterprise Tier Only) Enable Spring Cloud Gateway.')
+        c.argument('gateway_instance_count',
+                   arg_group="Spring Cloud Gateway",
+                   type=int,
+                   validator=validate_gateway_instance_count,
+                   is_preview=True,
+                   help='(Enterprise Tier Only) Number of Spring Cloud Gateway instances.')
+        c.argument('enable_api_portal',
+                   arg_group="API portal",
+                   arg_type=get_three_state_flag(),
+                   default=False,
+                   is_preview=True,
+                   help='(Enterprise Tier Only) Enable API portal.')
```

Comment:
Could we consider using `get_three_state_flag()` instead of `action='store_true'` ?

Reply:
Updated, thanks for suggestion!